### DISCUSSION TITLE
License GPL-3

### DISCUSSION THREAD

**Discussion by robHPE** [cheshire-cat-ai/core#698] [General]
Hello,
We are interested in using your framework for a complex project, but we have some concerns about the license you have chosen. The GPL-3 license is too restrictive for our purposes and may limit our ability to distribute our software. We kindly request that you consider changing the license to a more permissive one, such as MIT or Apache, or that you confirm that we can use the API of your framework with an application that has a different license. We appreciate your cooperation and look forward to hearing from you soon.

**Comment by pieroit** [comment]
Hello @robHPE 

1 - We are not going to change license, it's part of our core values.
2 - If you build around the Cat without modifying the core or the admin (both GPLv3), you are free to cover your product with the license you prefer.
3 - If you *edit* core or admin, and redistribute, you need to release your versions under GPLv3. Plugins can also have the license you prefer.
4 - We require that you contribute to the framework at least by linking on your product (both webpage and product itself) the "powered by Cheshire Cat AI" logo ([media kit](https://cheshirecat.ai/media-kit/)).
5 - Feel free to reach out and collab with us, opening a GH account just for this questions without joining the community and openly talk about your project is not flying well with the whole open ecosystem.

Welcome!
